README.md

```diff
@@ -168,9 +168,11 @@ html_theme_path = ['a-plus-rst-tools/theme']
 
 ### 1. Graded questionnaire
 
-The questionnaire directive arguments define the exercise key and max points
-with the optional difficulty. For example, `.. questionnaire:: 1 A50` sets key `1`,
-max points `50` and difficulty `A`.
+The questionnaire directive arguments define the exercise key and optional max points
+with the difficulty. For example, `.. questionnaire:: 1 A50` sets key `1`,
+max points `50` and difficulty `A`. If not set in the directive arguments, the max points will be set to
+the sum of the question points. Both difficulty and max points can be defined whether the other one is
+defined or not.
```

Comment:
"Both difficulty and max points can be defined whether the other one is defined or not."

I think this sentence is unclear and can be written in a better way. I assume you are saying that setting the difficulty is optional and the max points are taken from the sum of the question points if they are not manually defined in the argument.